### ISSUE TITLE
Gas prices probably need re-balancing

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#11286]
![image](https://user-images.githubusercontent.com/31366439/190293321-8994a7e5-2363-451b-89c9-83c202e72e34.png)

10-30% of that is okay depending on effort required but 150k is a bit much.

**Comment by Cheackraze** [comment]
I understand where youre coming from, but the time and effort it takes to get this normally is some of the most challenging technical atmos work there is. I achieved that in a 3 hour MRP extended round, with an artifact that generated _cold tritium_ with electricity use. The only other way to get this much in any reasonable way would be just a straight up frezon artifact. The higher prices for tritium and frezon was supposed to incentivize atmosians to do something other than set up distro and fuck off the rest of the round.

Here was the can in question:
![lots of frezon](https://user-images.githubusercontent.com/71046427/190301735-276d11b0-490e-4480-8241-ebbd5ab69ded.png)
![lots of frezon2](https://user-images.githubusercontent.com/71046427/190301753-4b06ee8d-1c73-49d3-ad6b-2d8c566a8f9f.png)

**Comment by retequizzle** [comment]
I see this more as an outlier than anything else. Tritium as a whole is a pain in the ass to synthesize normally, so the cost of tritium (and frezon by proxy) should be reflective of this. Artifacts as a whole are so RNG-based that to tweak entire game mechanics around one instance of it happening is overkill.

It merits watching to see how often a tritium artifact is actually secured and utilized like this, but that's about it. Either that, or just remove tritium as an artifact effect if you're worried about shortcuts being used to cheese cargo rounds.

**Comment by Partmedia** [comment]
Agreed. What happened here was that cargo got lucky and found a nice artifact *and* brought it to atmos *and* someone experienced was running atmos *and* synthesized a bunch of freezon. In a normal round it is not usually possible to synthesize nearly as much (typically in the <10,000 credit range) as was done here.

**Comment by Cheackraze** [comment]
> It merits watching to see how often a tritium artifact is actually secured and utilized like this, but that's about it. Either that, or just remove tritium as an artifact effect if you're worried about shortcuts being used to cheese cargo rounds.

9/10 times in LRP when they get a trit artifact its immediately followed by "OH SHIT CARGO IS ON FIRE"

**Comment by liltenhead** [comment]
Was gonna write a new issue, but this issue is still relevant and more of an issue than it used to be. Atmos techs can make hundreds of thousands of dollars in the first twenty or thirty minutes of a shift. It mostly invalidates the need for salvage to actually make money. One single atmos tech can make enough money to mindshield the entire station on their own. All of salvage working together would struggle to make that much money, even with cargo doing bounties.

**Comment by K-Dynamic** [comment]
Should be closed; we tend to bounce back and forth on gas prices anyway

**Comment by ArtisticRoomba** [comment]
for future ref, gas price balance is Okay:tm: but theres still core issues with atmos gas gameplay, mainly no diminishing returns
